subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -221,16 +221,16 @@ void lll_disable(void *param)
 int lll_prepare_done(void *param)
 {
 #if defined(CONFIG_BT_CTLR_LOW_LAT) && \
```

Comment:
Use  IS_ENABLED here also?

Reply:
ticker_op_job_disable is declared in a #if define (line:55), so in that case only CONFIG_BT_CTLR_LOW_LAT would be in an IS_ENABLED, and LLL_PRIO and ULL_LOW_PRIO would have to be an #if define. Considered it better to not mix the two.

Reply:
OK